### PR TITLE
Added elixir mentoring notes for beer song

### PR DESCRIPTION
**Pull request by chriseyre2000**
Added some basic notes on mentoring beer song.

### CODE REVIEW

File: tracks/elixir/exercises/beer-song/mentoring.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+### Reasonable solutions
+
+```elixir
```
Comment (SleeplessByte) [inline tracks/elixir/exercises/beer-song/mentoring.md]: If we compare this to the suggested solution from `99 bottles` (the book), this doesn't seem to be thé reasonable solution we're looking for?

File: tracks/elixir/exercises/beer-song/mentoring.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+def verse(0), do: ...
+```
+is clearer than
+```
+def  verse(number) when number == 0. do: ...
+```
+
+#### Style guide
+
+This is the first time on the track that students get to define their own functions and frequently trip over naming conventions.
```
Comment (SleeplessByte) [inline tracks/elixir/exercises/beer-song/mentoring.md]:
```suggestion
This is the first time on the track that students get to define their own functions and frequently trip over naming conventions. Refer them to the [elixer style guide](https://github.com/christopheradams/elixir_style_guide).
```

File: tracks/elixir/exercises/beer-song/mentoring.md (hunk 1 above)
Comment (chriseyre2000) [reply tracks/elixir/exercises/beer-song/mentoring.md]: It's a reasonable Elixir solution that maximises readability.

File: tracks/elixir/exercises/beer-song/mentoring.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+def verse(0), do: ...
+```
+is clearer than
+```
+def  verse(number) when number == 0. do: ...
+```
+
+#### Style guide
+
+This is the first time on the track that students get to define their own functions and frequently trip over naming conventions. Refer them to the [elixer style guide](https://github.com/christopheradams/elixir_style_guide).
```
Comment (SleeplessByte) [inline tracks/elixir/exercises/beer-song/mentoring.md]:
```suggestion
This is the first time on the track that students get to define their own functions and frequently trip over naming conventions. Refer them to the [elixir style guide](https://github.com/christopheradams/elixir_style_guide).
```

**Comment by SleeplessByte** [comment]
@exercism/website-copy-elixir please check correctness, then let's merge and further edits on new PRs

**Comment by rpottsoh** [comment]
Pinging @chriseyre2000.

**Comment by neenjaw** [comment]
As requested by @SleeplessByte in the #track-elixir channel on slack, I looked over the mentoring notes, they look reasonable and follows closely with how I mentor solutions.

**Comment by SleeplessByte** [comment]
Thank you! :rocket:

**Pull request merged**